debian/rules

```diff
@@ -21,6 +21,8 @@ DEB_SCONS_OPTIONS := --config=force --no-rpath USE_INSTALLED=all \
                      $(SCONS_ARGS)
 
 export GOCACHE := $(CURDIR)/.gocache
+# https://github.com/spdk/spdk/issues/2011
+export DEB_CFLAGS_MAINT_APPEND = $(shell pkg-config --cflags libdpdk)
```

Comment:
Is this still needed?

Reply:
no, will remove in a follow-up unless I have to repush

Reply:
removed

Reply:
Please don't resolve conversations started by reviewers.  It up to the reviewer that started the conversation to resolve it if/when their comments have been addressed sufficiently.